frame/contracts/src/schedule.rs

```diff
@@ -110,6 +110,13 @@ pub struct Limits {
 	pub code_size: u32,
 }
 
+impl Limits {
+	/// The maximum memory size in bytes that a contract can occupy.
+	pub fn max_memory_size(&self) -> u32 {
+		self.memory_pages * 64 * 1024
```

Comment:
This is 64 because we only support 64 bit machine ?

Reply:
Wasm memory pages are always 64KiB.